app/workers/dependency_resolution_worker.rb

```diff
@@ -13,7 +13,11 @@ def perform(args = {})
       send_downstream(content_id, locale)
     end
 
-    orphaned_content_ids.each { |content_id| send_downstream(content_id, locale) }
+    orphaned_content_ids_for_locale = Document.where(
+      content_id: orphaned_content_ids, locale: locale,
```

Comment:
Nitpick: I'd move `locale: locale` on its own line so it's easier to scan:

```suggestion
      content_id: orphaned_content_ids,
      locale: locale,
```

Reply:
Sure thing 👍